### ISSUE TITLE
Previous/Next Episode

### ISSUE THREAD

**Issue by Enteleform** [specfm/spec-next#926]
### Feature Request

Please add a way to navigate to the previous and next episodes from `https://spec.fm/podcasts/<podcast-name>/<episode-number>`.

### Problem/Example

I'm working through the `Developer Tea` podcast in its entirety, chronologically. My current workflow is:
- visit https://spec.fm/podcasts/developer-tea on my mobile device
- scroll with 14 full-out fling gestures to get to the end
  (*srs, I counted. also, sorting options would be useful here.*)
- scroll up while visually parsing for the episode I'm currently on
  (*doable for now, because the early episodes are numbered. probably would be pretty difficult to maintain once I hit the unnumbered episodes.*)
- click the episode title to access the episode page
- listen to the episode
- repeat the whole process next time I want to listen to an episode

### Improved Workflow
- bookmark current episode page
- listen to an episode
- click `next` button

**Comment by brianlovin** [comment]
Good idea! We might have to do some hackery to figure out next/previous from the api. Want to take a stab at this?
